semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -632,6 +632,7 @@ message SymbolInformation {
   Signature signature = 17;
   repeated Annotation annotations = 13;
   Access access = 18;
+  string documentation = 20;
```

Comment:
This can be removed. This file is the legacy version of the protocol. The file can honestly be removed, but no need to worry about that here.